docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -48,107 +37,89 @@ Follow these steps:
    ```
 
    :::{note}
-   This will automatically download the image if it does not exist on the host.
-   You can also pass the -v argument to mount any data directories from the host
-   onto the container.
+   This will automatically download the image if it does not exist on the host. You can also pass the `-v`
+   argument to mount any data directories from the host onto the container.
    :::
 
-(install_pytorch_using_wheels)=
+(install_pytorch_wheels)=
 
-#### Option 2: Install PyTorch Using Wheels Package
+## Using a wheels package
 
-PyTorch supports the ROCm platform by providing tested wheels packages. To
-access this feature, refer to
-[https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/)
-and choose the "ROCm" compute platform. {numref}`Installation-Matrix-from-Pytorch` is a matrix from <https://pytorch.org/> that illustrates the installation compatibility between ROCm and the PyTorch build.
+PyTorch supports the ROCm platform by providing tested wheels packages. To access this feature, go
+to [https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/). In the interactive
+table, choose ROCm from the _Compute Platform_ row.
 
-```{figure} ../../data/how_to/magma_install/image.006.png
-:name: Installation-Matrix-from-Pytorch
----
-align: center
----
-Installation Matrix from Pytorch
-```
+1. Choose one of the following three options:
 
-To install PyTorch using the wheels package, follow these installation steps:
+   **Option 1:**
 
-1. Choose one of the following options:
-   a. Obtain a base Docker image with the correct user-space ROCm version
-      installed from
-      [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
+   Download a base Docker image with the correct user-space ROCm version. See:
+   [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
```

Comment:
We can provide all of available ROCm images

| Base OS      | Docker image name | Link to Docker image |
| :---         | :---              | :----                |
| Ubuntu 20.04 | rocm/dev-ubuntu-20.04 | [https://hub.docker.com/r/rocm/dev-ubuntu-20.04](https://hub.docker.com/r/rocm/dev-ubuntu-20.04) |
| Ubuntu 22.04 | rocm/dev-ubuntu-22.04 | [https://hub.docker.com/r/rocm/dev-ubuntu-22.04](https://hub.docker.com/r/rocm/dev-ubuntu-22.04) |
| Centos 7     | rocm/dev-centos-7 | [https://hub.docker.com/r/rocm/dev-centos-7](https://hub.docker.com/r/rocm/dev-centos-7) |